ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -579,7 +579,9 @@ tests:
       ADDITIONAL_WORKERS: "1"
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "2"
+      MTO_OPERATOR_INSTALL_METHOD: catalog
       MTO_POD_PLACEMENT_API_VERSION: v1alpha1
+      SUBSCRIPTION_CHANNEL_OVERRIDE: main
```

Comment:
Perhaps add them in test chains, which will have less maintain effort IMO.

Reply:
what does it mean? adding parameter in chain file? do we have an example?